<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CGenerateSeismicRecordClass</class>
 <widget class="QDialog" name="CGenerateSeismicRecordClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>851</width>
    <height>817</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>生成合成地震记录</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_5">
   <item row="0" column="0" colspan="2">
    <widget class="QGroupBox" name="InputLayerParGroupBox">
     <property name="title">
      <string>输入地层参数</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="Par1Label">
        <property name="text">
         <string>第1层厚度(m)：</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QDoubleSpinBox" name="Par1DoubleSpinBox">
        <property name="decimals">
         <number>1</number>
        </property>
        <property name="maximum">
         <double>1000.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="Par2Label">
        <property name="text">
         <string>第1层速度(m/s)：</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QDoubleSpinBox" name="Par2DoubleSpinBox">
        <property name="decimals">
         <number>1</number>
        </property>
        <property name="maximum">
         <double>10000.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QLabel" name="Par16Label">
        <property name="text">
         <string>第1层密度(kg/m3)：</string>
        </property>
       </widget>
      </item>
      <item row="0" column="5">
       <widget class="QDoubleSpinBox" name="Par16DoubleSpinBox">
        <property name="decimals">
         <number>1</number>
        </property>
        <property name="maximum">
         <double>10000.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="Par3Label">
        <property name="text">
         <string>第2层厚度(m)：</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QDoubleSpinBox" name="Par3DoubleSpinBox">
        <property name="decimals">
         <number>1</number>
        </property>
        <property name="maximum">
         <double>1000.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="Par4Label">
        <property name="text">
         <string>第2层速度(m/s)：</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QDoubleSpinBox" name="Par4DoubleSpinBox">
        <property name="decimals">
         <number>1</number>
        </property>
        <property name="maximum">
         <double>10000.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="1" column="4">
       <widget class="QLabel" name="Par17Label">
        <property name="text">
         <string>第2层密度(kg/m3)：</string>
        </property>
       </widget>
      </item>
      <item row="1" column="5">
       <widget class="QDoubleSpinBox" name="Par17DoubleSpinBox">
        <property name="decimals">
         <number>1</number>
        </property>
        <property name="maximum">
         <double>10000.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="Par5Label">
        <property name="text">
         <string>第3层厚度(m)：</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QDoubleSpinBox" name="Par5DoubleSpinBox">
        <property name="decimals">
         <number>1</number>
        </property>
        <property name="maximum">
         <double>1000.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QLabel" name="Par6Label">
        <property name="text">
         <string>第3层速度(m/s)：</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QDoubleSpinBox" name="Par6DoubleSpinBox">
        <property name="decimals">
         <number>1</number>
        </property>
        <property name="maximum">
         <double>10000.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="2" column="4">
       <widget class="QLabel" name="Par18Label">
        <property name="text">
         <string>第3层密度(kg/m3)：</string>
        </property>
       </widget>
      </item>
      <item row="2" column="5">
       <widget class="QDoubleSpinBox" name="Par18DoubleSpinBox">
        <property name="decimals">
         <number>1</number>
        </property>
        <property name="maximum">
         <double>10000.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="Par7Label">
        <property name="text">
         <string>第4层厚度(m)：</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QDoubleSpinBox" name="Par7DoubleSpinBox">
        <property name="decimals">
         <number>1</number>
        </property>
        <property name="minimum">
         <double>-100.000000000000000</double>
        </property>
        <property name="maximum">
         <double>1000.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QLabel" name="Par8Label">
        <property name="text">
         <string>第4层速度(m/s)：</string>
        </property>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="QDoubleSpinBox" name="Par8DoubleSpinBox">
        <property name="decimals">
         <number>1</number>
        </property>
        <property name="maximum">
         <double>10000.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="3" column="4">
       <widget class="QLabel" name="Par19Label">
        <property name="text">
         <string>第4层密度(kg/m3)：</string>
        </property>
       </widget>
      </item>
      <item row="3" column="5">
       <widget class="QDoubleSpinBox" name="Par19DoubleSpinBox">
        <property name="decimals">
         <number>1</number>
        </property>
        <property name="maximum">
         <double>10000.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="Par9Label">
        <property name="text">
         <string>第5层厚度(m)：</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QDoubleSpinBox" name="Par9DoubleSpinBox">
        <property name="decimals">
         <number>1</number>
        </property>
        <property name="maximum">
         <double>1000.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QLabel" name="Par10Label">
        <property name="text">
         <string>第5层速度(m/s)：</string>
        </property>
       </widget>
      </item>
      <item row="4" column="3">
       <widget class="QDoubleSpinBox" name="Par10DoubleSpinBox">
        <property name="decimals">
         <number>1</number>
        </property>
        <property name="maximum">
         <double>10000.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="4" column="4">
       <widget class="QLabel" name="Par20Label">
        <property name="text">
         <string>第5层密度(kg/m3)：</string>
        </property>
       </widget>
      </item>
      <item row="4" column="5">
       <widget class="QDoubleSpinBox" name="Par20DoubleSpinBox">
        <property name="decimals">
         <number>1</number>
        </property>
        <property name="maximum">
         <double>10000.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="Par11Label">
        <property name="text">
         <string>第6层厚度(m)：</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QDoubleSpinBox" name="Par11DoubleSpinBox">
        <property name="decimals">
         <number>1</number>
        </property>
        <property name="maximum">
         <double>1000.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="5" column="2">
       <widget class="QLabel" name="Par12Label">
        <property name="text">
         <string>第6层速度(m/s)：</string>
        </property>
       </widget>
      </item>
      <item row="5" column="3">
       <widget class="QDoubleSpinBox" name="Par12DoubleSpinBox">
        <property name="decimals">
         <number>1</number>
        </property>
        <property name="maximum">
         <double>10000.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="5" column="4">
       <widget class="QLabel" name="Par21Label">
        <property name="text">
         <string>第6层密度(kg/m3)：</string>
        </property>
       </widget>
      </item>
      <item row="5" column="5">
       <widget class="QDoubleSpinBox" name="Par21DoubleSpinBox">
        <property name="decimals">
         <number>1</number>
        </property>
        <property name="maximum">
         <double>10000.000000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QGroupBox" name="RickerWaveletParGroupBox">
     <property name="title">
      <string>输入雷克子波参数</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="3">
       <widget class="QLabel" name="Par14Label">
        <property name="text">
         <string>△t（ms）：</string>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QDoubleSpinBox" name="Par14DoubleSpinBox">
        <property name="decimals">
         <number>1</number>
        </property>
        <property name="maximum">
         <double>100.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="Par13Label">
        <property name="text">
         <string>子波主频(Hz)：</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QDoubleSpinBox" name="Par13DoubleSpinBox">
        <property name="decimals">
         <number>1</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QGroupBox" name="RandomNoisePercentGroupBox">
     <property name="title">
      <string>输入随机噪音参数</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="0">
       <widget class="QLabel" name="Par15Label">
        <property name="minimumSize">
         <size>
          <width>175</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>噪音强度(%)：</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QDoubleSpinBox" name="Par15DoubleSpinBox">
        <property name="minimumSize">
         <size>
          <width>176</width>
          <height>0</height>
         </size>
        </property>
        <property name="decimals">
         <number>1</number>
        </property>
        <property name="maximum">
         <double>1000.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>输出文件</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="OutputFile1Label">
        <property name="text">
         <string>输出雷克子波b(t)：</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="2">
       <widget class="QLineEdit" name="OutputFile1LineEdit"/>
      </item>
      <item row="0" column="3">
       <widget class="QPushButton" name="OutputFile1PushButton">
        <property name="text">
         <string>. . .</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="OutputFile2Label">
        <property name="text">
         <string>输出随机噪音n(t)：</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="2">
       <widget class="QLineEdit" name="OutputFile2LineEdit"/>
      </item>
      <item row="1" column="3">
       <widget class="QPushButton" name="OutputFile2PushButton">
        <property name="text">
         <string>. . .</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="OutputFile3Label">
        <property name="text">
         <string>输出地层反射系数ξ(t)：</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1" colspan="2">
       <widget class="QLineEdit" name="OutputFile3LineEdit"/>
      </item>
      <item row="2" column="3">
       <widget class="QPushButton" name="OutputFile3PushButton">
        <property name="text">
         <string>. . .</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="OutputFile4Label">
        <property name="text">
         <string>输出合成地震记录x(t)：</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1" colspan="2">
       <widget class="QLineEdit" name="OutputFile4LineEdit"/>
      </item>
      <item row="3" column="3">
       <widget class="QPushButton" name="OutputFile4PushButton">
        <property name="text">
         <string>. . .</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="OutputFile5Label">
        <property name="text">
         <string>输出雷克子波振幅谱B(k)：</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1" colspan="2">
       <widget class="QLineEdit" name="OutputFile5LineEdit"/>
      </item>
      <item row="4" column="3">
       <widget class="QPushButton" name="OutputFile5PushButton">
        <property name="text">
         <string>. . .</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0" colspan="2">
       <widget class="QLabel" name="OutputFile6Label">
        <property name="text">
         <string>输出无噪音地震记录振幅谱S(k)：</string>
        </property>
       </widget>
      </item>
      <item row="5" column="2">
       <widget class="QLineEdit" name="OutputFile6LineEdit"/>
      </item>
      <item row="5" column="3">
       <widget class="QPushButton" name="OutputFile6PushButton">
        <property name="text">
         <string>. . .</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0" colspan="2">
       <widget class="QLabel" name="OutputFile7Label">
        <property name="text">
         <string>输出含噪音地震记录振幅谱X(k)：</string>
        </property>
       </widget>
      </item>
      <item row="6" column="2">
       <widget class="QLineEdit" name="OutputFile7LineEdit"/>
      </item>
      <item row="6" column="3">
       <widget class="QPushButton" name="OutputFile7PushButton">
        <property name="text">
         <string>. . .</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QPushButton" name="CalPushButton">
     <property name="text">
      <string>计算</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QPushButton" name="CancelPushButton">
     <property name="text">
      <string>取消</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="generate_seismic_record.qrc"/>
 </resources>
 <connections/>
</ui>
